src/components/MovieForm/MovieForm.css

```diff
@@ -78,3 +78,65 @@
 .submit-button:hover {
   opacity: 0.8;
 }
+
+.react-select-container {
+  width: 100%;
+  font-size: 14px;
+}
+
+.react-select__control {
+  background-color: black !important;
```

Comment:
I know why you did here important!
But for the future, better to avoid !important

Reply:
you can use CSS Specificity